src/style.css

```diff
@@ -0,0 +1,122 @@
+:root {
+  --color-blue: #00acdc;
+  --color-white: #fff;
+  --color-black: #060b35;
+  --color-gray: #616070;
+  --color-milk: #f3f3f3;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  background: var(--color-white);
+  border: 1px solid var(--color-milk);
+  border-radius: 5px;
+  padding: 16px;
+}
+
+.card__image {
+  max-width: 160px;
+  margin: 16px auto 0;
+  display: block;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-black);
+  margin: 40px 0 4px;
```

Comment:
it's recommended to use one type for adding an outer indent between elements. Either margin-bottom or margin-top. Apply all over the page
Example:
```
.image {
  margin: 0 auto 10px;
}

.title {
  margin-bottom: 16px;
}

.code {
  margin-bottom: 8px;
}
```